src/Misc/layoutbin/en-US/strings.json

```diff
@@ -197,6 +198,8 @@
     "bin/Agent.Listener configure --url http://mylocaltfsserver:8080/tfs --agent myagent --pool default --nostart --acceptteeeula --auth negotiate --username DEV-BOX\\developer --password MyPassword"
   ],
   "ListenForJobs": "{0:u}: Listening for Jobs",
+  "LocalClockSkewed": "Lock clock might skew too great, please sync with internet time server, time skew must be less than 5 minutes.",
+  "LockSkewStopRetry": "Stop retry on VssOAuthTokenRequestException after retried for {0} seconds.",
```

Comment:
The local machine's clock may be out of sync with the server time by more than five minutes. Please sync your clock with your domain or internet time and try again.

Stopped retrying OAuth token request exception after {0} seconds.